linera-service/Cargo.toml

```diff
@@ -48,7 +48,7 @@ kubernetes = [
     "dep:fs_extra",
 ]
 remote-net = ["dep:k8s-openapi", "dep:kube"]
-metrics = ["prometheus", "linera-base/metrics"]
+metrics = ["prometheus", "linera-base/metrics", "linera-client/metrics"]
```

Comment:
`linera-client` already includes them:

```toml
metrics = [
    "linera-base/metrics",
    "linera-chain/metrics",
    "linera-core/metrics",
    "linera-execution/metrics",
    "linera-rpc/metrics",
    "linera-views/metrics",
]
```

Maybe `linera-base` doesn't want to be there either to be honest :thinking: I left it in just because it was already there.

Reply:
Oh I think you're right, I put it there in the first place, and that was because `linera-base/metrics` is required to compile with metrics support (because if metrics is enabled we call things that are conditionally compiled).